3604/1/create_rating.rb

```diff
@@ -0,0 +1,36 @@
+# class for create rating
+class CreateRating
+  def initialize(array_gems)
+    @array_gems = array_gems
+    @array_coefficient = [1, 0.5, 1, 0.7, 0.5, 0.1]
+  end
+
+  def create_rating
+    priority_counting
+    sort_by_value
+    @array_gems
```

Comment:
not needed here.
And better use attr_readers (even private) instead of instance variables. It's much easier to refactor and better fits object-oriented design principles.